src/main/java/com/chariotsolutions/meetupnotifier/EventIdRow.java

```diff
@@ -0,0 +1,20 @@
+package com.chariotsolutions.meetupnotifier;
+
+public class EventIdRow {
+  private String googleId;
+  private String meetupId;
+
+  public EventIdRow(String googleId, String meetupId) {
+    this.googleId = googleId;
```

Comment:
I would add assertions (preconditions) here that check the strings are not empty.  You can look into Google Guava's set of preconditions - https://code.google.com/p/guava-libraries/wiki/PreconditionsExplained